UnitTests/Utilities/GeometryModelRenderer/src/BinVisualTest.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <BinVisualTest.h>
+#include <boost/filesystem.hpp>
+
+using namespace testing;
+
+void BinVisualTest::SetUp()
+{
+	rendererOff = buw::makeReferenceCounted<OffGeometryModelRenderer>(renderSystem_);
```

Comment:
I'm a bit confused here since you created your on GeometryModelRenderer for BIN but are still using the OffGeometryModelRenderer
Maybe because some unit test can just reuse the OffGeometryModelRenderer and others can't?
I'm not sure if get this right

Reply:
I created GeometryModelRenderer for BIN to see which options I will have. I also created  GeometryModelRenderer for PointCloud. I will clean and update the code, thus it will function right as soon as I understand, how I will be implementing it. I also delete redundant code to avoid confusion

Reply:
Removed BIN converter from OIP here: https://github.com/tumcms/Open-Infra-Platform/commit/ec928809fbeafcaa757bf3c0a80abaa3321a5489